src/utils/getNormalizedStateData.js

```diff
@@ -48,12 +48,12 @@ const getNormalizedStateData = (data, stateCode) => {
     if (item.state_code === stateCode) {
       const normalizedItem = {
         metric: item.metric,
-        year: parseInt(item.year, 10),
-        month: parseInt(item.month, 10) - 1,
+        year: parseInt(item.year),
```

Comment:
> Just removing the 10 arg won't have the desired behavior because [the radix argument does not default to 10](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Syntax).

Good point! I didn't know that.